src/io/flutter/run/daemon/DaemonEvent.java

```diff
@@ -227,7 +240,8 @@ void accept(Listener listener) {
 
     private Boolean finished;
 
-    @NotNull String getType() {
+    @NotNull
+    String getType() {
       return progressId == null ? "" : progressId;
```

Comment:
FWIW: `StringUtil.notNullize(progressId);` is another cute/idiomatic way to do this.

Reply:
👍, updated